packages/lumx-core/src/scss/components/thumbnail/_index.scss

```diff
@@ -137,3 +145,20 @@
         @include lumx-state(lumx-base-const('state', 'FOCUS'), lumx-base-const('emphasis', 'LOW'), 'light');
     }
 }
+
+/* Thumbnail badge mask
+   ========================================================================== */
+
+@each $key, $size in $lumx-sizes {
```

Comment:
Can we remove that part? Not sure it's needed? It's going to be weird with light badges -> it has box shadow